style.css

```diff
@@ -44,7 +44,8 @@ body {
 .no-signal {
   width: 100%; 
   height: 100%; 
-  position: relative;
+  position: fixed; /* changed to fixed */
+  z-index: -1;
```

Comment:
Really found this interesting when you tested the z-index which kept the background continuing down with the list ✨ was super kool